model/participant/Participant.ttl

```diff
@@ -102,6 +112,38 @@ ids:participantRefinement
     rdfs:comment "Conditions which needs to be satisfied that a single Participant can be seen as a member of the subject Participant. For instance, all Participants with their headquarter in Europe might be a potential consumer of GDPR-releated data."@en;
 .
 
+ids:hasBusinessIdentifier
+	a owl:ObjectProperty ;
+	rdfs:domain ids:Participant ;
+	rdfs:range ids:BusinessIdentifier ;
+	rdfs:label "has business identifier"@en;
+	rdfs:comment "Company identifiers, described via ids:BusinessIdentifier."@en ;
+	.
+	
+ids:vatID
```

Comment:
Please be consistent with the property naming. Either add "hasProperty" or "property" to all (cf. `hasBusinessIdentifier` vs `vatID`), thanks!

Reply:
Also see `legalForm` and `jurisdiction` below.